sticker-utils/src/bin/sticker-train.rs

```diff
@@ -51,6 +53,12 @@ impl TrainApp {
 impl TrainApp {
     fn new() -> Self {
         let matches = sticker_app("sticker-train")
+            .arg(
+                Arg::with_name(CONTINUE)
+                    .long("continue")
+                    .value_name("PARAMS")
+                    .help("Continue training from a parameter file"),
```

Comment:
maybe change help message to "Continue training from parameter file specified in config"?

Reply:
But it is not continuing from the parameter file specified in config ;). It uses the parameter file that is given as an argument to `--continue`.

Reply:
Just assumed this because I didn't recognize that `.value_name()` implicitly sets `.takes_value(true)`, but I guess that should be expected since a "value_name" doesn't make sense without an argument.

Reply:
Might still make sense to add some info like appending "e.g. epoch-50" to the help message since each epoch produces two files and it might be unclear to users what is the "parameter file"

Reply:
> Just assumed this because I didn't recognize that `.value_name()` implicitly sets `.takes_value(true)`, but I guess that should be expected since a "value_name" doesn't make sense without an argument.

I will add it for explicitness. The clap docs do it as well.